src/Servers/Kestrel/test/InMemory.FunctionalTests/Http2/Http2StreamTests.cs

```diff
@@ -2681,7 +2727,6 @@ public async Task WriteAsync_GetMemoryLargeWriteBeforeFirstFlush()
             await InitializeConnectionAsync(async httpContext =>
             {
                 var response = httpContext.Response;
-                await response.StartAsync();
```

Comment:
If the test isn't explicitly about omitting StartAsync, and calling StartAsync is more efficient, call StartAsync.